tests/scenarios/scenarios.ts

```diff
@@ -42,6 +33,12 @@ async function lts_3_24(project: Project) {
   project.linkDevDependency('@ember/string', { baseDir: __dirname });
 }
 
+async function lts_3_28(project: Project) {
+  project.linkDevDependency('ember-source', { baseDir: __dirname, resolveName: 'ember-source-3.28' });
+  project.linkDevDependency('ember-cli', { baseDir: __dirname, resolveName: 'ember-cli-3.28' });
+  project.linkDevDependency('ember-data', { baseDir: __dirname, resolveName: 'ember-data-3.38' });
```

Comment:
It seems like this typo is the cause of the CI faillures.
```suggestion
  project.linkDevDependency('ember-data', { baseDir: __dirname, resolveName: 'ember-data-3.28' });
```

Reply:
Thanks, good catch, I had fired this off and didn't come back around to look at why it was failing yet.